includes/tools.h

```diff
@@ -12,6 +11,7 @@
 #endif
 #ifdef STM32F4
 #include "stm32f4xx_hal.h"
+#include "stm32f429xx.h"
```

Comment:
Rozumiem intencję, ale tego typu includy powinny być w każdym pliku. Po to są te __NAME_H__, żeby kompilator się nie powtórzył. Ale my jako programiści chcemy mieć wprost pokazane że ta biblioteka, w tej sytuacji, korzysta z tego. Do "tools" można wyciągać ogólniki, jak stdint